mypy/fastparse.py

```diff
@@ -219,6 +219,7 @@ def translate_module_id(self, id: str) -> str:
         if id == self.custom_typing_module:
             return 'typing'
         elif id == '__builtin__' and self.pyversion[0] == 2:
+            assert False  # Shouldn't get here
```

Comment:
These asserts are just for my peace of mind -- I think when @Michael0x2a forked fastparse.py into fastparse2.py, he left some version-checking code in that could be reduced because we know it's always Python 2. (For stubs and type comments, the Python 3 syntax is always parsed with fastparse.py.)